users/models.py

```diff
@@ -46,17 +49,27 @@ def create_superuser(self, email, first_name,
             raise ValueError('superuser must be assigned to is_superuser=True')
 
         return self.create_user(email, first_name,
-                                last_name, birth_date, password,
+                                last_name, birth_date, password, user_type,
                                 **other_fields)
 
 
+class UserType(models.TextChoices):
```

Comment:
Correct me if I'm wrong, isn't it better this class will be a subclass in User?

Reply:
I believe Owner/Seeker choice can be handy in other places as well, not just the user class. Our code will be loaded with features that are different for seekers and owners. That's why I decided to keep it flexible and outside. Open for other opinions as well!